COVID19/src/utils/functions/mainFunctions.js

```diff
@@ -0,0 +1,27 @@
+const getFixedDate = () => {
+    let date = new Date();
+    let day = date.getDate();
+    let month = date.getMonth() + 1;
+    let year = date.getFullYear();
+    let fullDate = '';
+    if (month < 10) {
+      fullDate = `${day}/0${month}/${year}`;
```

Comment:
You can make a function to format this value and you can recycle

Reply:
The indicated code is precisely part of the getFixedDate function, is that what you mean?